tests/jballerina-unit-test/src/test/resources/test-src/expressions/typecast/type_cast_expr.bal

```diff
@@ -378,7 +378,28 @@ function testFutureCastPositive() returns boolean {
 function testFutureCastNegative() {
     future<int> s1 = start testFutureFunc();
     any a = s1;
-    future<int[]> s2 = <future<int[]>> a;
+    future<int[]>|error s2 = trap <future<int[]>> a;
+
+    assertEquality(true, s2 is error);
```

Comment:
Shall we use `test:assertEquals` in all places as this is a unit test. In unit tests we are able to import `ballerina\test`

Reply:
Addressed with the commit https://github.com/ballerina-platform/ballerina-lang/pull/30960/commits/49019610515d02faf157bfbeec8ab86a9da21cf8